### PR TITLE
Translate tray icon context menu

### PR DESCRIPTION
**Pull request by ArkadiuszMichalski**
Fix https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11628.

@donho 
This node is necessary here?
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/235b51af67de112c86b5e0a522b586d8dc826832/PowerEditor/installer/nativeLang/english.xml#L388-L389

### CODE REVIEW

File: PowerEditor/src/NppBigSwitch.cpp [hunk 1]
```diff
@@ -2597,6 +2597,7 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 					HMENU hTrayIconMenu;  // shortcut menu
 					hmenu = ::LoadMenu(_pPublicInterface->getHinst(), MAKEINTRESOURCE(IDR_SYSTRAYPOPUP_MENU));
 					hTrayIconMenu = ::GetSubMenu(hmenu, 0);
+					_nativeLangSpeaker.changeLangTryIconContexMenu(hTrayIconMenu);
```
Comment (chcg) [inline PowerEditor/src/NppBigSwitch.cpp:2600]: typo ...TryIcon... instead of ...TrayIcon...

**Comment by donho** [comment]
@ArkadiuszMichalski 
> This node is necessary here?

No. you can remove it.

**Comment by donho** [comment]
@ArkadiuszMichalski 
Oups, Sorry for the conflict.
Could you solve it please?

**Comment by donho** [comment]
@ArkadiuszMichalski 
Thank you!

**Pull request closed without merging**